File: 2091-removing-minimum-and-maximum-from-array/2091-removing-minimum-and-maximum-from-array.py
class Solution:
    def minimumDeletions(self, nums: List[int]) -> int:
        
        max_index = nums.index(max(nums))
        min_index = nums.index(min(nums))

        if max_index < min_index:
            max_index, min_index = min_index, max_index
        

        return min (
            max(min_index, max_index) + 1,
            len(nums) - min(min_index, max_index),
            min_index + 1 + len(nums) - max_index,
        )

File: 2091-removing-minimum-and-maximum-from-array/2091-removing-minimum-and-maximum-from-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minimumDeletions = function(nums) {
    let max_index = nums.indexOf(Math.max(... nums));
    let min_index = nums.indexOf(Math.min(... nums));

    if (min_index > max_index) {
        let total = min_index + max_index;
        min_index = total - min_index;
        max_index = total - max_index;
    }

    return Math.min(
        max_index + 1,
        nums.length - min_index,
        min_index + 1 + nums.length - max_index
    );
};